airbyte-workers/src/main/java/io/airbyte/workers/temporal/TemporalUtils.java

```diff
@@ -66,7 +66,8 @@ public static WorkflowServiceStubs createTemporalService(final String temporalHo
   private static final Configs configs = new EnvConfigs();
   public static final RetryOptions RETRY = RetryOptions.newBuilder()
       .setMaximumAttempts(configs.getActivityNumberOfAttempt())
-      .setInitialInterval(Duration.ofSeconds(configs.getDelayBetweenActivityAttempts()))
+      .setInitialInterval(Duration.ofSeconds(configs.getInitialDelayBetweenActivityAttemptsSeconds()))
+      .setMaximumInterval(Duration.ofSeconds(configs.getMaxDelayBetweenActivityAttemptsSeconds()))
```

Comment:
This means that the maximum interval between attempt retries will default to 10 minutes, which prevents the exponential backoff from causing attempts to have hours of delay between retries which would slow down automatic system recovery.

These new settings mean that activities will be retried with the following logic:
```
Attempt 1 fails
(wait 30s)
Attempt 2 fails
(wait 1m)
Attempt 3 fails
(wait 2m)
Attempt 4 fails
(wait 4m)
Attempt 5 fails
(wait 8m)
Attempt 6 fails
(wait 10m)
Attempt 7 fails
(wait 10m)
Attempt 8 fails
(wait 10m)
Attempt 9 fails
(wait 10m)
Attempt 10 fails
(execute workflow restart logic)
```
Adding up all those above times gives us a total timespan of 55.5 minutes over which an activity will be retried 10 times, before the workflow is restarted.

Reply:
Where is the logic that increases the wait time between failed activity attempts?

Reply:
I'm thinking we should lower the retries down to 5, which gives us about 15 mins until a workflow is restarted. 

55 mins seems like a long one restart cycle loop.

Reply:
I think 5 retries makes sense - I will make that change

Reply:
I have pushed a change to update the default value for this `ACTIVITY_MAX_ATTEMPT` env var from 10 to 5